src/api-handlers/__tests__/fixtures/mines.fixture.ts

```diff
@@ -0,0 +1,114 @@
+import { MineDocument, Mine, MineResult, MinesResponse, MinesSearchQuery, MinesRequestQueryOptions } from '../../mines';
+
+export const sampleMineDocument: MineDocument = {
+  _id: '61fd3a3bcc6d81005d135799',
+  miner: '4uzia.wam',
+  params: {
+    invalid: 0,
+    error: '',
+    delay: 560,
+    difficulty: 0,
+    ease: 69,
+    luck: 6,
+    commission: 95,
+  },
+  bounty: 837,
+  land_id: '1099512958747',
+  planet_name: 'neri.world',
+  landowner: 'xkubg.wam',
+  bag_items: [1099575138963, 1099571561418, 1099572783734],
+  offset: 125,
+  block_num: 165058267,
+  block_timestamp: {
+    date: '2022-02-04T14:34:53Z',
+  },
+  global_sequence: 37593403954,
+  tx_id: '91f0f074f841cafba129c28f65e551f022f7a2b5ba2fdc3463291fc7aee29ce1',
+};
+
+export const sampleMine: Mine = {
+  id: '61fd3a3bcc6d81005d135799',
+  miner: '4uzia.wam',
```

Comment:
do you know if we should obfusicate out all these out...they are like PII to me...although to be honest they are also on the public blockchain ...up to you

Reply:
changed and actually that should be one of the points in our how to write tests to use only fake data for unit tests or our internal for e2e tests, not clients/users etc